e2e/suites/navigation/sidebar.test.ts

```diff
@@ -151,10 +151,10 @@ describe('Sidebar', () => {
 
   it('[C217152] File Libraries tooltip', async () => {
     await page.clickFileLibraries();
-    expect(await sidenav.getLinkTooltip(SIDEBAR_LABELS.FILE_LIBRARIES)).toContain('File Libraries');
```

Comment:
this should remain unchanged. This refers to the sidenav link, which is called File Libraries, the parent of My Libraries and Favorite Libraries